cafebabel/articles/drafts/views.py

```diff
@@ -17,8 +17,8 @@ def list():
 
 
 @drafts.route('/new/', methods=['get', 'post'])
-@editor_required
 @login_required
+@editor_required
```

Comment:
Not fond of the fact that order matters here, maybe add automagically `login_required` when `editor_required` is added? It looks more consistent to me.

Reply:
What is not found?
I'm ok to have `login_required` embedded into `editor_required` in a different PR for keeping its original purpose on this one.

Reply:
Without the U? https://www.vocabulary.com/dictionary/fond

OK to put it in a subsequent PR

Reply:
Issue [opened here](https://github.com/cafebabel/cafebabel.com/issues/114) for memory.